### PR TITLE
Document MempoolCapacityBytesOverride

### PR DESCRIPTION
**Pull request by newhoggy**
Document `MempoolCapacityBytesOverride` in the example mainnet YAML configuration file.

### CODE REVIEW

File: configuration/cardano/mainnet-config.yaml [hunk 1]
```diff
@@ -263,3 +263,11 @@ options:
 hasPrometheus:
   - 127.0.0.1
   - 12798
+
+# Set or unset the mempool capacity override in number of bytes.
+#
```
Comment (dcoutts) [inline configuration/cardano/mainnet-config.yaml:268]:
```suggestion
# Set or unset the mempool capacity override in number of bytes.
#
# This is intended for testing, and for low-resource machines to run with a smaller mempool.
# Please note that running with a large mempool is NOT recommended. The mempool is a
# just a network communication buffer and all the advice on "buffer bloat" applies, see:
# https://en.wikipedia.org/wiki/Bufferbloat
# The default size is two blocks, which is generally enough to ensure full blocks.
#
```

Comment (newhoggy) [reply configuration/cardano/mainnet-config.yaml:268]: Done!

**Comment by newhoggy** [comment]
bors merge

**Comment by dcoutts** [comment]
bors merge

**Review by Jimbo4350 — APPROVED** [review]

**Comment by benapetr** [comment]
is there more documentation somewhere?

I would like to understand:
* if this is not recommended to be higher than default, then what is the point of this option?
* What happens if I make the buffer higher than default?
* Why would it cause problems? Bufferbloat is problem with UDP video streams and similar, cardano is different scenario
* What happens if this is made smaller than default?
* Why is default only 2 blocks?

**Pull request merged**